manifests/plugin/remote_execution/ssh.pp

```diff
@@ -49,6 +52,18 @@
     template_path => 'foreman_proxy/plugin/remote_execution_ssh.yml.erb',
   }
 
+  if $ssh_kerberos_auth {
+    if $::osfamily == 'RedHat' and $::operatingsystem != 'Fedora' {
+      $scl_prefix = 'tfm-'
+    } else {
+      $scl_prefix = '' # lint:ignore:empty_string_assignment
+    }
+
+    foreman_proxy::plugin { 'net-ssh-krb':
```

Comment:
I think you should use ```package``` since this isn't a plugin, it's a library. Another thing is that this will break on Debian where the package would most likely be named ```ruby-net-ssh-krb``` but isn't packaged.